receiver/riakreceiver/metadata.yaml

```
@@ -0,0 +1,70 @@
+name: riakreceiver
+
+attributes:
+  riak.node.name:
+    description: The name this node uses to identify itself.
+  request.operation:
+    value: state
+    description: The request operation type.
+    enum:
+      - put
+      - get
+  index.operation:
```

```suggestion
  operation:
```

Keep in mind, when someone is looking at this label, they are doing so in the context of a metric called "Index Operation Count", so the index part is understood.